TheFourFifths/consus-client
# View all items tff35

### Summary
RUN WITH SERVER ON BRANCH: item-data-tff88
REVIEW THIS VERY HEAVILY. There are so many changes we should bump this to version 2.0. Heres wha ha happened:

- You can view all items
- You can view the model of an item
- API calls were simplified
- Models and Items now generate properly by using components.
- Styl files are all similarly formatted. 

### Checklist


Finally, is it ready to be reviewed and merged: :green_apple: 

### How to Review

1. DO EVERYTHING
2. Checkout an item
3. Checkin an item
4. View all items
5. Create an item
6. Verify item is created and shows in the view all items screen
7. Create a model
8. Verify model is created and shows in the view all models screen
9. Check a student can't checkout something if overdue item or w/e

### Links

[Server PR](https://github.com/TheFourFifths/consus/pull/27)
